playbooks/group_vars/grafana_servers/20-ldap_config.yaml

```
@@ -0,0 +1,34 @@
+# vim:ts=2:sw=2:et:ai:sts=2
+---
+
+grafana_ldap__servers__host: "{{ (glauth__service_ldap.listen.split(':')[0]
+                                  if glauth__service_ldap.listen is defined else None) }}"
+grafana_ldap__servers__port: "{{ (glauth__service_ldap.listen.split(':')[1]
+                                  if glauth__service_ldap.listen is defined else None) }}"
+grafana_ldap__servers__use_ssl: "{{ (glauth__service_ldap.tls | lower
+                                    if glauth__service_ldap.tls is defined else None) }}"
+grafana_ldap__servers__start_tls: "{{ (glauth__service_ldap.tls | lower
+                                      if glauth__service_ldap.tls is defined else None) }}"
+grafana_ldap__servers__ssl_skip_verify: 'false'
+grafana_ldap__servers__bind_dn: "cn=grafana,
+                                  dc={{ host_fqdn.split('.')[-3] }},
+                                  dc={{ host_fqdn.split('.')[-2] }},
+                                  dc={{ host_fqdn.split('.')[-1] }}"
+# FIXME: hacer coincidir el pwd con el hash bcryp en glauth
+grafana_ldap__servers__password: 'eiw3CueL'
```

this should be placed in the secrets store
```suggestion
grafana_ldap__servers__password: "{{ lookup('community.general.passwordstore', 'ldap/grafana')}}"
```

---

But also, this should be in the client repo too